server/lib/pbench/common.py

```diff
@@ -0,0 +1,2 @@
+tarball = "server/lib/pbench/test/fixtures/upload/tarball/log.tar.xz"
+bad_tarball = "server/lib/pbench/test/fixtures/upload/bad_tarball/bad_log.tar.xz"
```

Comment:
Shouldn't this file be under the `server/lib/pbench/test` directory to clarify it's used by the test system?

Reply:
yes, I will move it there